docs/docs/03-data-engineering-roadmap.md

```diff
@@ -23,3 +23,81 @@ title: Data Engineering Roadmap
 ## Overview
 
 [!["Roadmap"](https://github.com/datastacktv/data-engineer-roadmap/raw/master/img/roadmap.png)](https://github.com/datastacktv/data-engineer-roadmap)
+
+### Software Engineering
+
+อาชีพวิศวกรซอฟแวร์เป็นอีกหนึ่งอาชีพที่มีความใกล้ชิดกับอาชีพวิศวกรข้อมูลเป็นอย่างมาก เพราะหนึ่งในสกิลที่เชื่อมโยงทั้งสองอาชีพอยู่เรียกว่า **วิศวกรรมซอฟแวร์ (Software Engineering)** หรือสกิลการเขียนโปรแกรมซึ่งเป็นอีกหนึ่งสกิลที่ Data Engineer ควรมีติดตัวไว้ เนื่องจากเป็นอาชีพที่ต้องจัดการกับข้อมูลขนาดใหญ่จึงต้องศึกษาเกี่ยวกับประสิทธิภาพในการออกแบบฐานข้อมูล และแพลตฟอร์มต่าง ๆ ด้วย
+
+จากหัวข้อ [Data Engineer Archtypes](https://www.thedataengineeringbook.online/docs/data-engineer#data-engineer-archtypes) ได้มีการพูดถึงประเภทของ Data Engineer ซึ่งแบ่งตามประสบการณ์ และสกิลที่ใช้ในการทำงานในแต่ละองค์กร เช่น Data Engineer ที่ใช้ SQL ในการออกแบบ database และวิเคราะห์ข้อมูล หรือแม้แต่ตำแหน่ง GUI-based solutions ที่ใช้เครื่องมือประเภท No-code ในการจัดการกับข้อมูลก็อาจได้ใช้สกิลด้านโปรแกรมมิ่งน้อยกว่า Data Engineer ในสายอื่น ๆ ดังนั้น สกิลโปรแกรมมิ่งจึงจำเป็นกับสายงาน Data Engineer แต่ควรเลือกเรียนรู้ตามบริบทในแต่ละองค์กรนั่นเอง
```

Comment:
ตรง link นี่เราสามารถใช้เป็น internal link ได้เปล่านะครับ แบบที่ไม่ต้องใส่ full URL @kahnwong

Reply:
yes we can def use internal link 😉 https://docusaurus.io/docs/markdown-features/links

Reply:
@kahnwong Cool!

@firstnapat I guess this is our last request for this PR. 😂 Please change the link.